### PR TITLE
20 backsetup apollo

### PR DESCRIPTION
**Pull request by Cavonstavant**
### Follow up update on db setup

This pr replace the use of a mongodb by postgreSQL

It also includes simple way to setup your db with prisma and apollo using a justfile
See here for more detail on the just command launcher [https://github.com/casey/just](https://github.com/casey/just)

Here are the current reciepies implemented:
```bash
> just -l
Available recipes:
    prisma_migrate env # Migrate database schema to desired DB with specified environment (possible values are: prod, preprod and dev)
    prisma_schema_gen  # Generate a new prisma schema that will be used inside the code
    prisma_studio env  # Launch and open prisma studio in specified environment (possible values are: prod, preprod and dev)
```

- orm: Migrate to postgresql and add uuid to `User` model
- build: remove bson as it's not needed anymlore
- misc: add migrations folder to gitignore
- dev: seed db only in dev mode
- build: Add justfile and update readme

### CODE REVIEW

**Review by Bootoyka — COMMENTED** [review]
OK i'll update & try your pr in mine using docker-compose

File: database/src/config/seed.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { PrismaClient, Prisma } from '@prisma/client';
+import { env } from 'process';
+const prisma = new PrismaClient();
+
+const dummyUserData : Prisma.UserCreateInput[] = [
```
Comment (Bootoyka) [inline database/src/config/seed.ts:5]: #0 3.854 src/config/seed.ts(5,30): error TS2694: Namespace '"/usr/src/app/node_modules/.prisma/client/index".Prisma' has no exported member 'UserCreateInput'.

**Review by Bootoyka — COMMENTED** [review]
pls fix seed.ts

File: database/src/config/seed.ts (hunk 1 above)
Comment (Bootoyka) [reply database/src/config/seed.ts:5]: cf : 
```
const user = await prisma.user.create({
  data: {
    email: 'elsa@prisma.io',
    name: 'Elsa Prisma',
  },
})
```

**Review by EmileEttel — APPROVED** [review]
lgtm !

**Pull request merged**